src/js/common/schemaform/widgets/SelectWidget.jsx

```diff
@@ -49,7 +49,7 @@ function SelectWidget({
           const newValue = getValue(event, multiple);
           onChange(processValue(schema, newValue));
         }}>
-      <option value="">{placeholder}</option>
+      {!schema.default && <option value="">{placeholder}</option>}
```

Comment:
This is the primary change. This is what the non-customized SelectWidget does in rjsf.